sqladmin/models.py

```diff
@@ -448,6 +448,18 @@ class UserAdmin(ModelView, model=User):
     Unlimited by default.
     """
 
+    custom_actions_in_list: ClassVar[Dict[str, str]] = {}
```

Comment:
If I understand correctly these are only for internal configurations, not for setup by user.
In that case you can just set them up in `__init__` with `self._ custom_actions_in_list ` to mark internal.

Reply:
yeah it make sense to move it into ModelView as private